routes/api/gameApi.js

```diff
@@ -1,11 +1,29 @@
 const router = require("express").Router();
 
 const Games = require("../../models/Games");
+const Users = require("../../models/Users");
+const GameCollection = require("../../models/GameCollection");
 const { withAuth } = require("../../utils/auth");
 
 // find all games for the user
-router.get("/", withAuth, async (req, res) => {});
+router.get("/id", withAuth, async (req, res) => {
+  try {
+    const userGames = await Games.findByPk(req.params.id, {
+      include: [{ model: Users, through: GameCollection, as: "user_games" }],
```

Comment:
have you tested this?

Reply:
removing from this branch as i am working on it in a different one now